catalog/spatial/registry/registry-admin-modules/registry-admin-poller-service/src/main/java/org/codice/ddf/registry/admin/poller/AdminPollerServiceBeanMBean.java

```diff
@@ -25,15 +25,15 @@
 public interface AdminPollerServiceBeanMBean {
     boolean sourceStatus(String servicePID);
 
-    List<Map<String, Object>> allSourceInfo();
+    List<List<Map<String, Object>>> allSourceInfo();
```

Comment:
Should add a brief java doc to these methods so we can know what to expect from them